buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -2281,10 +2281,12 @@ public void run() {
                         byte[] array = new byte[8];
                         buf.readBytes(array);
 
+                        //noinspection MisorderedAssertEqualsArguments
```

Comment:
As I understand `MisorderedAssertEqualsArguments` triggering warning on the `assertEquals` and `assertArrayEquals` if the first argument is not a constant. I can't found better description.. Only this [commit to IDEA](http://git.jetbrains.org/?p=idea/community.git;a=commit;h=638b01d6de0e53bf29ccd72b1d6ee6ed707009b5).
Just keep as it is without any suppressions?